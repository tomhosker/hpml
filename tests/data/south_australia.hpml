In South Australia I was born,
##MINICHORUS Heave away! Haul away!
In South Australia round #PLACE{Cape Horn}.
##MINICHORUS We're bound for South Australia!

###CHORUS
Haul away, you rolling king!
Heave away! Haul away!
Haul away! O hear me sing:
We're bound for South Australia!

As I walked out one morning fair,
There I met Miss #PERSON{Nancy Blair}.

There ain't but one thing grieves my mind:
To leave Miss #PERSON{Nancy Blair} behind.

I ran her all night; I ran her all day,
Ran her before we sailed away.

I shook her up; I shook her down;
I shook her round #ADD round #ADD round.

O when we lollop around #PLACE{Cape Horn},
##MINICHORUS Heave away! Haul away!
You'll wish to God you'd never been born.
##MINICHORUS We're bound for South Australia!

###CHORUS
Haul away, you rolling king!
Heave away! Haul away!
Haul away! O hear me sing:
We're bound for South Australia!
