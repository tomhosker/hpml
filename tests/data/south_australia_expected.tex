\settowidth{\versewidth}{You'll wish to God you'd never been born.}
\begin{verse}[\versewidth]
In South Australia I was born,\\*
\vin \textit{Heave away! Haul away!}\\
In South Australia round \textsc{Cape Horn}.\\*
\vin \textit{We're bound for South Australia!}\\!

{\itshape
Haul away, you rolling king!\\
Heave away! Haul away!\\
Haul away! O hear me sing:\\*
We're bound for South Australia!}\\!

As I walked out one morning fair,\\*
There I met Miss \textit{Nancy Blair}.\\!

There ain't but one thing grieves my mind:\\*
To leave Miss \textit{Nancy Blair} behind.\\!

I ran her all night; I ran her all day,\\*
Ran her before we sailed away.\\!

I shook her up; I shook her down;\\*
I shook her round \& round \& round.\\!

O when we lollop around \textsc{Cape Horn},\\*
\vin \textit{Heave away! Haul away!}\\
You'll wish to God you'd never been born.\\*
\vin \textit{We're bound for South Australia!}\\!

{\itshape
Haul away, you rolling king!\\
Heave away! Haul away!\\
Haul away! O hear me sing:\\*
We're bound for South Australia!}
\end{verse}
